mjcf_to_sdformat/mjcf_to_sdformat/converters/sensor.py

```diff
@@ -22,6 +22,48 @@
 import sdformat_mjcf_utils.sdf_utils as su
 
 CAMERA_INDEX = 0
+IMU_INDEX = 0
+
+
+def mjcf_accelerometer_gyro_sensor_to_sdf(sensor, model):
+    """
+    Converts a accelerometer or gyro sensor from MJCF to SDFormat.
+    :param mjcf.sensor sensor: The MJCF accelerometer or gyro sensor to convert
+    :param sdf.Model sensor: The MJCF sensor to convert
+    :return: The newly created SDFormat IMU sensors.
+    :rtype: [sdf.Sensor]
```

Comment:
```suggestion
    :rtype: sdf.Sensor
```
Since it's not returning a list.

Reply:
https://github.com/gazebosim/gz-mujoco/pull/66/commits/85d9fc198e8d58c35c66eea80c358da77f7f73ab